modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/render.es.js

```diff
@@ -62,7 +62,11 @@ export default function render(renderable, renderData, container) {
 			}
 		);
 
-		const Component = typeof renderable === 'function' ? renderable : null;
+		const Component =
+			typeof renderable === 'function' ||
+			renderable.$$typeof === Symbol.for('react.forward_ref')
```

Comment:
Seems to do the trick, @wincent... was this what you meant?

Reply:
Sadly, yes.

Reply:
("Sadly" because I worry that we might end up with 20 `||` branches for this kind of special case one day. But I don't think we have any real choice.)

Reply:
Let's hope it doesn't come to that!